ajax/libs/jquery.scrollspeed/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "jquery.scrollspeed",
+  "filename": "jQuery.scrollSpeed.min.js",
+  "description": "Extension for custom scrolling speed and easing",
+  "version": "1.0.2",
+  "author": "Nathan Rutzky",
+  "homepage": "https://code.nath.co/scrollSpeed",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/brightcove/videojs-thumbnails.git"
```

Comment:
The repo's URL is not the current lib which be added by you.
Please modify this.
Thanks